airflow/serialization/base_serialization.py

```diff
@@ -123,6 +122,27 @@ def _is_excluded(cls, var, attrname, instance):
             cls._value_is_hardcoded_default(attrname, var)
         )
 
+    @classmethod
+    def serialize_to_json(cls, object_to_serialize: Union[BaseOperator, DAG], decorated_fields: Set):
+        """Serializes an object to json"""
+        serialized_object = {}
+        keys_to_serialize = object_to_serialize.get_serialized_fields()
+        for key in keys_to_serialize:
+            # None is ignored in serialized form and is added back in deserialization.
+            value = getattr(object_to_serialize, key, None)
+            if cls._is_excluded(value, key, object_to_serialize):
+                continue
+
+            if key in decorated_fields:
+                serialized_object[key] = cls._serialize(value)
+            else:
+                value = cls._serialize(value)
+                # TODO: Why?
```

Comment:
Why?

Reply:
Where did this fn come from? It looks like a code change?

Reply:
Oh it was shared from both S10nOperator and S10nDag. Gotcha.

Reply:
Let's remove this comment, one version has it and the other doesn't, and I don't think it adds anything.

Reply:
Yes, this can be removed! Was from PR5743

Reply:
Yeah.

Reply:
Yeah.